### ISSUE TITLE
about train code

### ISSUE THREAD

**Issue by 85zhanghao** [Knightzjz/NCL-IML#6]
Hi, thank you for your awesome work. Do you have any plans to release the training code?

**Comment by Knightzjz** [comment]
Hi Zhang, thanks for your nice question.  
  Dozens of similar queries reached Me and our corresponding author, Prof. Feng, these days. So, we would like to leverage this chance here to deliver an official answer towards researchers holding the same doubt.  
  We, the IML-scu group, are investigating heavily into the IML field and trying everything to push the frontiers of base IML models. In other words, we are conducting a series of research aiming to address the IML problem rather than publishing several papers. Hence, the open-source schedule for our work is carefully planned in order to keep our research pace. The timeline for training code release is arranged in the following order:  
  [IML-Vit](https://github.com/SunnyHaze/IML-ViT)(already released)->IML-ViTv2(ETR: Dec 2023, same repo in IML-Vit)->[PMAE](https://arxiv.org/pdf/2310.06525.pdf)(ETR: Mar 2024)->NCL-IML (ETR: June 2024)->[MMG](https://github.com/ndyysheep/MTVNR)(Manipulation Mask Generator, ETR: June 2024)  
Some of the code releases, particularly IML-ViTv2, may be far ahead of the paper. We will be informing everyone through this repo. Stay tuned.
